neo/core/regionofinterest.py

```diff
@@ -0,0 +1,172 @@
+# -*- coding: utf-8 -*-
+
+from math import floor, ceil
+
```

Comment:
A general and stupid question : this RegionOfInterest is binary by design (in or out) but we could also have a weighted region of interest. No ?

Reply:
Yes, that would make sense for circular regions of interest (e.g. circular gaussian weighting). I'm not sure how you would specify it for a more complex shape, however.